packages/vue/src/components/molecules/SfDropdown/SfDropdown.vue

```diff
@@ -77,15 +82,18 @@ export default {
     },
   },
   methods: {
+    open() {
+      this.$emit("click:open");
+    },
     close() {
       this.$emit("click:close");
     },
     checkPersistence() {
-      if (!this.persistent) this.close();
+      if (!this.persistent) return this.$emit("click:close");
```

Comment:
Why did you decide to replace close method?

Reply:
In fact, I don't know :)  It should stay like it was.